### ISSUE TITLE
Is it maintained?

### ISSUE THREAD

**Issue by notramo** [amberframework/amber#1290]
I have read the discussion about the project being abandoned by the main developer.
What's the status of the project now? Is it maintained?
Is it production-ready? Can I use it for more serious projects?

**Comment by elorest** [comment]
Amber core works and is _somewhat_ maintained, although I've only put about 30 hours into it this year. I know others have addressed a few issues as well. Integration with some third party libraries has been the primary challenge. In some cases I've had to fork them into amber and update them to work with the latest version of crystal and require them in shard.yml instead of the original library. Back in 2017 @eliasjpr @drujensen myself and a few hours put thousands of hours into this project, but have since been forced to spend less time on by work and life. I'm aware of a few groups using it in production and historically it has worked. As new versions of crystal come out it may require a bit of extra effort to setup a new project.

TLDR; I'm to maintaining core functionality don't have much time for new functionality or integrations with third party DSLs etc. If you have time and ideas lets talk.

**Comment by crimson-knight** [comment]
@notramo I am actively maintaining Amber, and working on enhancements.

Yes you could use Amber in production and serious applications. Please join the discord group if you want updates and to participate in shaping the future of Amber.

I hope to start publishing screencasts in October.

**Comment by notramo** [comment]
@crimson-knight I'm glad that this project isn't abandoned. If I get familiar with it, I might contribute some enhancements too.
Unfortunately, I don't use Discord because of this: [Please don't use Discord for FOSS projects](https://drewdevault.com/2021/12/28/Dont-use-Discord-for-FOSS.html)
Do you plan to bridge the Discord room to some FOSS platform? [Matrix](https://matrix.org) is a good choice for open source projects, and there are already publicly hosted bridges, which can be used by anyone: https://t2bot.io/discord/

**Comment by crimson-knight** [comment]
@notramo I have no plans to use any other platforms at this point. I will eventually expand the main amber website to include a forum where more people can interact and seek the community, but that's down the line.
